lxml-stubs/builder.pyi

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Callable, Mapping, Optional
+
+from .etree import _AnyStr, _Element
+
+class ElementMaker:
+    def __init__(
+        self,
+        typemap: Optional[Mapping[Any, Callable[[_Element, Any], None]]] = ...,
+        namespace: Optional[str] = ...,
+        nsmap: Optional[Mapping[Any, _AnyStr]] = ...,

```

Comment:
I think we should assume str:str mappings for nsmap.
```suggestion
        nsmap: Optional[Mapping[_AnyStr, _AnyStr]] = ...,

```

Reply:
Hm, done some experiment, it's not the case. Currently implementation allows default NS in ElementMaker construction, [see this gist](https://gist.github.com/abelcheung/078cf2ca35e73bc13f39644f47864468).

That said, I'd prefer str:str as well for any `nsmap` arguments not accepting default NS. Don't have any statictics here, but is it OK to assume the user case of using byte key is negligible to be discarded? If that's true, I'd want to do some global cleanup in future.

Reply:
Looks like `_AnyStr` is a Union, so it's better to avoid it in a key type, because the key type of Mapping is invariant. That means that if the user passes a `dict[str, str]`, it won't type check.

Reply:
@scoder, how about going ahead first? Guess I should come up with a `Mapping` replacement (as per suggestion further above) with next PR or so, that properly address similar problem once and for all.